test/studies/shootout/binary-trees/binarytrees-pool.chpl

```diff
@@ -8,31 +8,31 @@
 
 use Allocators;
 
-config const n = 10,         // the maximum tree depth
-             globalPoolSize = (2 ** 31)-1,
-             localPoolSize = globalPoolSize;
-
-var globalPool = new bumpPtrMemPool(globalPoolSize, alignment=0);
+config const n = 10;                         // the maximum tree depth
 
 proc main() {
-  const minDepth = 4,                      // the shallowest tree
-        maxDepth = max(minDepth + 2, n),   // the deepest normal tree
-        strDepth = maxDepth + 1,           // the depth of the "stretch" tree
-        depths = minDepth..maxDepth by 2;  // the range of depths to create
-  var stats: [depths] (int,int);           // stores statistics for the trees
+  const minDepth = 4,                        // the shallowest tree
+        maxDepth = max(minDepth + 2, n),     // the deepest normal tree
+        strDepth = maxDepth + 1,             // the depth of the "stretch" tree
+        depths = minDepth..maxDepth by 2,    // the range of depths to create
+        nodeSize = 24,                       // the approximate size of a node
```

Comment:
I'd still drop "approximate" here because it suggests uncertainty; and if we're wrong, it's almost certainly too small.

Reply:
That change got left behind in my buffer :(, fixed now